### PR TITLE
Update participants.json

### PR DESCRIPTION
**Pull request by cuber15**
\\Saloni Sharma updated as participant

### CODE REVIEW

File: Data/participants.json [hunk 1]
```diff
@@ -7,5 +7,11 @@
       "facebookLink": "https://www.facebook.com/arhumk1",
       "githubLink": "https://github.com/arhum602"
     }
```
Comment (dojutsu-user) [inline Data/participants.json:9]:
```suggestion
    },
```

File: Data/participants.json [hunk 2]
```diff
@@ -7,5 +7,11 @@
...
-  ]
+    {
+  "name": "Saloni Sharma",
+      "email": "lcs2019053@iiitl.ac.in",
+      "profilePicture": "https://avatars1.githubusercontent.com/u/57322099?s=460&v=4",
+      "facebookLink": "https://www.facebook.com/profile.php?id=100009442653201",
+      "githubLink": "https://github.com/cuber15"
+}]
```
Comment (dojutsu-user) [inline Data/participants.json:16]:
```suggestion
        }
    ]
```

**Review by dojutsu-user — CHANGES_REQUESTED** [review]

**Comment by dojutsu-user** [comment]
Please don't make PR from your `master` branch. This is not considered as a good practice.
Make a separate branch and then make a PR from that.

**Comment by rishabhshuklax** [comment]
There's #27 already opened for the same issue, kindly do not open multiple PRs for the same issue.

**Pull request closed without merging**